src/ortolan.ts

```diff
@@ -0,0 +1,57 @@
+import {exec} from '@actions/exec'
+
+type Repository = {
+    owner: string
+    repo: string
+}
+type Repositories = Repository[]
+type RunType = 'local-path' | 'list-of-repositories'
+
+export class Ortolan {
+    private repositories: Repositories
+
+    constructor(repositories: Repositories) {
+        this.repositories = repositories
+    }
+
+    analyze(owner: string, repo: string): void {
+        const cwd = process.cwd()
+        const bindMountInput = `${cwd}/in/${owner}/${repo}:/project`
+        const bindMountOutput = `${cwd}/out/${owner}/${repo}:/out`
+        exec(
+            `docker run --rm -v ${bindMountInput} -v ${bindMountOutput} ort analyze -i /project -o /out`
```

Comment:
Make CI green by giving publicly available image name
```suggestion
        const bindMountOutput = `${cwd}/out/${owner}/${repo}:/out`
        const image = 'philipssoftware/ort'
        exec(
            `docker run --rm -v ${bindMountInput} -v ${bindMountOutput} ${image} analyze -i /project -o /out`
```